pallets/metamask-signature/src/benchmarking.rs

```diff
@@ -0,0 +1,35 @@
+//! Benchmarking setup for pallet-template
+#![cfg(feature = "runtime-benchmarks")]
+use super::*;
+
+#[allow(unused)]
+use crate::Pallet as Template;
```

Comment:
Do we really want these template module stuff here? benches, mock and tests...?

Reply:
actually we dont need them :+1: